extensions/wikia/ExactTargetUpdates/tests/ExactTargetApiTest.php

```diff
@@ -60,30 +60,6 @@ function testPrepareSoapVarsShouldReturnSoapVarsArray() {
 	}
 
 	/**
-	 * @dataProvider deleteSubscriberProvider
-	 */
-	function testDeleteSubscriber( $aApiParams, $expectedRequest) {
-		// Mock tested class
-		$mockApiSubscriber = $this->getMockBuilder( 'Wikia\ExactTarget\ExactTargetApiSubscriber' )
-			->disableOriginalConstructor()
-			->setMethods( [ 'sendRequest' ] )
-			->getMock();
-		$mockApiSubscriber
-			->expects( $this->once() )
-			->method( 'sendRequest' )
-			->with( 'Delete', $expectedRequest );
-
-		// Set helper property
-		$oReflection = new ReflectionClass($mockApiSubscriber);
-		$oReflectionProperty = $oReflection->getProperty('helper');
-		$oReflectionProperty->setAccessible(true);
-		$oReflectionProperty->setValue($mockApiSubscriber, new Wikia\ExactTarget\ExactTargetApiHelper());
-
-		// Run tested method
-		$mockApiSubscriber->deleteRequest($aApiParams);
-	}
-
-	/**
```

Comment:
This test was in fact testing ```prepareSubscriberObjects``` method of ```\Wikia\ExactTarget\ExactTargetApiHelper```. What do you think about adding unit tests for that method?